src/components/Footer/Footer.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import className from '../../resources/vendors/css/Grid.css';
+import classes from './Footer.module.css';
+
+
+const footer = (props) => {
+    return (
+        <div className={classes.footer}>
+            <footer className="row section">
+                <div className="row box">
+                    <section className="col span-1-of-6">
+                        <div>
+                            <h3 className={classes.text}>DOD Digital Learning Portal</h3>
+                        </div>
+                    </section>
+                    <section className="col span-2-of-6">
+                        <div className="row">
+                            <h4 className={classes.text}>Links</h4>
+                            <ul className="col span-1-of-3">
+                                <li className={classes.listLinks}><a href="https://dodcio.defense.gov/" target="_blank" className={classes.links}>Home</a></li>
```

Comment:
I get a warning for each link element. I think we need rel="noreferrer" for all links.